pkg/deploy/saveversion.go

```diff
@@ -50,9 +35,17 @@ func (d *deployer) SaveVersion(ctx context.Context) error {
 		return err
 	}
 
+	d.log.Infof("instantiating blobs client using SAS token to upload content")
+	containerUrl := fmt.Sprintf("https://%s.blob.%s/%s", *d.config.Configuration.RPVersionStorageAccountName, d.env.Environment().StorageEndpointSuffix, "$web")
+	uploadBlobsClient, err := azblob.NewBlobsClientUsingEntra(containerUrl, tokenCredential, d.env.Environment().ArmClientOptions())
+	if err != nil {
+		d.log.Errorf("failure to instantiate blobs client using SAS: %v", err)
+		return err
+	}
+
```

Comment:
why is there a separate blobs client.  Can we not use the previous one?  Is there something special about the new service URL containing the $web container, even though we reference it in `upload buffer`